swagger.yaml

```diff
@@ -223,28 +253,114 @@ definitions:
       gap:
         description: "Mandatory gaps in trail (must be cleared)"
         type: "boolean"
-      
+
   Trail:
     type: "object"
     required:
     - "name"
     - "location"
     properties:
-      id:
-        type: "integer"
-        format: "int64"
-      name:
-        type: "string"
-        example: "Dan's Trail"
-      difficulty:
-        type: "string"
-        enum:
-        - "Green"
-        - "Blue"
-        - "Black"
-        - "Double Black"
-      features:
-        $ref: "#/definitions/Features"
-      location:
-        description: "Zip code of trail head"
-        type: "integer"
\ No newline at end of file
+      data:
+        properties:
```

Comment:
My bad if my previous comment didn't make sense about the attributes. Per jsonapi spec, you still need an attributes section for things like zipcode, name, etc. It should look like something like:
```
{
  "data": {
    "id": 0,
    "type": "trails",
    "attributes": {
      "name": "Dan's Trail",
      "zipCode": 0,
      "difficulty": "Green",
      "largeDrop": true,
      "smallDrop": true,
      "woodRide": true,
      "skinny": true,
      "largeJump": true,
      "smallJump": true,
      "gap": true
    }
  }
}
```
see: http://jsonapi.org/format/#fetching-resources-responses-200